src/ray/object_manager/object_store_pool.h

```diff
@@ -0,0 +1,62 @@
+#ifndef RAY_OBJECT_STORE_POOL_H
+#define RAY_OBJECT_STORE_POOL_H
+
+#include <list>
+#include <memory>
+#include <mutex>
+#include <vector>
+
+#include <boost/asio.hpp>
+#include <boost/asio/error.hpp>
+#include <boost/bind.hpp>
+
+#include "plasma/client.h"
+#include "plasma/events.h"
+#include "plasma/plasma.h"
+
+#include "object_directory.h"
+#include "ray/id.h"
+#include "ray/status.h"
+
+namespace ray {
+
+class ObjectStorePool {
```

Comment:
Document the class with `\class ObjectStorePool`. Also, I would maybe call this something like `ObjectStoreClientPool`.